minigalaxy/ui/gametile.py

```diff
@@ -553,7 +556,7 @@ def __state_queued(self):
         self.__create_progress_bar()
 
     def __state_downloading(self):
-        self.button.set_label(_("downloading…"))
+        self.button.set_label(_("Downloading…"))
```

Comment:
This change makes it necessary to update all translation files. So we'd either have to leave this change out or update those.